src/bundler.ts

```diff
@@ -191,14 +218,20 @@ class Bundler {
 
     const imprt = importMap.get(resolvedUrl);
     if (imprt) {
+      // If the document wasn't loaded for the import during analysis, we can't
+      // inline it.
+      if (!imprt.document) {
+        // TODO(usergenic): What should the behavior be when we don't have the
+        // document to inline available in the analyzer?
+        return;
```

Comment:
I'm very curious when this even happens. Is it an expected situation when running across external imports, say? For now maybe console a debug would be good, but eventually it'd be nice to have a more clearly meaningful condition to check on, especially so we can throw an error if a document doesn't load that we _should_ inline.